docs/getting-started/add-sample-data.asciidoc

```diff
@@ -1,22 +1,23 @@
 [[add-sample-data]]
 == Get up and running with sample data
 
-{kib} has three sample data sets that you can use to explore {kib} before loading your own data 
```

Comment:
I change from `three` to a more generic `several` because future sample data sets may be added and it will be easier to keep the docs in sync with the product if the docs are not so specific.